.eslintrc.js

```diff
@@ -1,16 +1,18 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 2017,
     sourceType: 'module',
   },
-  plugins: ['ember'],
+  plugins: ['ember', '@typescript-eslint', 'prettier'],
```

Comment:
Does including the plugin without enabling any of the rules do anything?

Reply:
Yeah, it turns out that including the plugin without setting any specific rules is what makes `yarn lint:s --fix` work for prettier fixes.

Reply:
TIL